woodwork/logical_types.py

```diff
@@ -788,3 +793,15 @@ def _validate_not_negative(series, return_invalid_values):
         if any_invalid:
             info = f"Series {series.name} contains negative values."
             raise TypeValidationError(info)
+
+
+def _coerce_string(series):
```

Comment:
What's the purpose of casting to a `string` before coercing to `numeric`? Is it to support boolean transformations?

Reply:
`pandas.to_numeric` doesn't work properly and raises an error with object types